### PR TITLE
Adv illegal ballistic Second Try!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]: # Adds in a illegal mags/clips to be made via Rnd for Sec to be able to make spare clips for guns like M1911!

Adds in a new tec node called Adc Illegal Basllisics
New node is locked and hidden, needing Ballistics node, Illegal tec node and Explosive Weapons node
Adds in the M1911 .45 Mags
Adds in Blot action stripper clip
Adds in all the 10mm Mags
Adds in 9mm mags
All only buildable in Sec

[why]: # For sec as well as traitors/space looters to be able to in theory get spare clips for there weapons

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
I flat out dont know what the problem is, its nothing to do with the code I added in but with engeryblasters shoot guns things. 
I need help with this >_<

File: modular_citadel/code/modules/research/techweb/all_nodes.dme.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Poojawa) [inline modular_citadel/code/modules/research/techweb/all_nodes.dme.dm]: This seems like an extra thing.

**Review by Poojawa — COMMENTED** [review]
modular_citadel/code/modules/projectiles/guns/pumpenergy.dm:15:error: bad argument definition

is where Travis is catching. Make sure your branch is fully updated.

File: modular_citadel/code/modules/research/techweb/all_nodes.dme.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/research/techweb/all_nodes.dme.dm]: It is, don't know were it came form sadly it was just in the clone. Didn't even touch it

Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/research/techweb/all_nodes.dme.dm]: I don't know why pumpenergy.dm is having a problem... I didn't even touch the files or anything!
Its just broken for me. Tried everything I could think if to fix the argument, sadly I lack knowing and such to fix it

**Review by deathride58 — CHANGES_REQUESTED** [review]
Fix the compiling errors.

The IDs of the designs do not match with the techweb node's design ids list.

As for the content of the PR, I'm really not sure how I feel about being able to print incendiary and HP clips without spending any TC at all. Incendiary and hollow point are both considerably strong ammo types. Adding the ability to print those two ammo types without spending any TC is a pretty significant buff to the stechkin.

**Comment by Trilbyspaceclone** [comment]
Well, its balanced in the fact it takes over a hour to get, needs illegal basic to unlock as well as sec access and have gotten a gun that can fire the ammo

**Comment by Poojawa** [comment]
Over an hour to get, when you can max research in 20m... ??

**Comment by Trilbyspaceclone** [comment]
Unless my math is wrong it takes 75,000(63 mins) points, as well as unlocking illegal tec. 
Thats if you 100% only get the nodes you need in order for you to be able to unlock it
Also you would have to get into sec and print those, easying could make you sus if not confirmed if the AI or any sec sees you making illegal ammo.

File: modular_citadel/code/modules/research/techweb/all_nodes.dme.dm (hunk 1 above)
Comment (silicons) [reply modular_citadel/code/modules/research/techweb/all_nodes.dme.dm]: delete this file

File: tgstation.dme [hunk 2]
```diff
@@ -2846,6 +2846,7 @@
 #include "modular_citadel\code\modules\projectiles\boxes_magazines\ammo_boxes.dm"
 #include "modular_citadel\code\modules\projectiles\boxes_magazines\external\smg\smg.dm"
 #include "modular_citadel\code\modules\projectiles\bullets\bullets\smg.dm"
+#include "modular_citadel\code\modules\projectiles\boxes_magazines\external\pistol.dm.dm"
```
Comment (silicons) [inline tgstation.dme]: needs to be .dm

File: modular_citadel/code/modules/projectiles/boxes_magazines/external/pistol.dm.dm [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/datum/design/m10mm/fire
```
Comment (silicons) [inline modular_citadel/code/modules/projectiles/boxes_magazines/external/pistol.dm.dm]: needs to be .dm, no extra .dm

**Comment by silicons** [comment]
forcing recompile

**Comment by Trilbyspaceclone** [comment]
Idea form now. Ima slowly remove everthing and see if its all gone, and still fail or not

**Comment by Trilbyspaceclone** [comment]
Now time to readd them all

**Comment by Trilbyspaceclone** [comment]
oh... welp I fucked that up...

**Comment by Trilbyspaceclone** [comment]
The Office season 7 ep 19 14:45

**Pull request closed without merging**